src/jobs/generation/MetaGenerator.groovy

```diff
@@ -371,7 +371,7 @@ repos.each { repoInfo ->
 
                         if (isPRTest) {
                             if (isVSTS) {
-                                // TODO: VSTS PR refspec
+                                refspec('+refs/pull/${ghprbPullId}/merge:refs/remotes/pull/${ghprbPullId}/merge')
```

Comment:
Same comment as above (re: ghprbPullId)

Reply:
As discussed, changed it to vstsRefspec, which is set by TFS Jenkins plugin depending on whether this is is a PR build.